migrate_pim/migrate_to_saas.rst

```diff
@@ -0,0 +1,141 @@
++--------------------------------+---------------------------------------------------------------------------------------+
+
+Disable custom code
+~~~~~~~~~~~~~~~~~~~
+
+Serenity does not support custom code within the PIM. In Serenity, all custom features use API connections and apps from the `App Store <https://apps.akeneo.com>`_ 
+(including `custom apps <https://api.akeneo.com/apps/create-custom-app.html>`_).
+
+**If you have custom code bundles, please** `remove them <https://docs.akeneo.com/master/maintain_pim/first_aid_kit/index.html#step-10-did-you-customize-your-pim>`_ 
+**before you begin the migration process.**
+
+If you cannot find a suitable replacement for your custom code, please contact your Customer Success Manager — they may be able to suggest changes
+or offer solutions that are compatibile with Serenity.
+
+.. warning::
+
+    We cannot validate your instance's migration to Serenity without all of the following information.
+    If we request any changes, we may ask you to run some commands again to check the updated status of your PIM.
+
+Check your database schema
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Please send us the results of the following shell commands so that we can ensure your database schema is up-up-to-date and compatible with a migration.
+
+.. code:: bash
+
+    $ cd /home/akeneo/pim
+    $ bin/console pimee:database:inspect -f --env=dev
+    $ composer require jfcherng/php-diff
+    $ bin/console pimee:database:diff --env=dev
+    $ bin/console doctrine:migrations:status
+
+.. warning::
+
+    If the results of ``bin/console doctrine:migrations:status`` show a non-zero number of "New" migrations, your
+    database is not up to date. Please run ``bin/console doctrine:migrations:migrate``, run the ``status`` command again,
+    and send us the output of all of these commands so that we can see any changes.
+
+Check your filesystem
+~~~~~~~~~~~~~~~~~~~~~
+
+We also require the output of the following commands, so that we can check to make sure your filesystem adapter is set up correctly for the migration to Serenity.
+
+.. code:: bash
+
+    $ cd /home/akeneo/pim
+    $ bin/console debug:config OneupFlysystemBundle
+
+Schedule a timeslot for migration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Akeneo Support team **must validate** the output of the above commands before we can schedule any migration.
+
+Once we have received all of this information and validated it, we can schedule the Flexibility to Serenity migration operation with our Cloud Engineering team. 
+Their hours are **Monday to Friday, 9:00 am to 6:00 pm CET**. Please provide at least 48 hours notice between your request and the actual migration time slot.
+This gives our team time to prepare and to ensure that your migration runs smoothly.
+
+For most instances, migrations will take between 2 and 4 hours. However, the duration of individual migration operations can vary — especially if your catalog is very large or complex.
```

Comment:
Could we say that a catalogue is "very large"? I'm afraid that they only think about the "number of products". We usually talking about "volumetry", as it could be on different axis

Reply:
I don't want to confuse anyone, and if a catalog is not large in terms of SKUs, but it does have many axes, I think that is covered by the phrase `or complex`.

Quantifying the complexity of a catalog and its effects is difficult, but the point of this document is not to make an estimate of how long the migration will last -- just to lay out general expectations. If a migration takes more than 4 hours, it is almost certainly because the catalog (axes, SKUs, etc -- inclusive) is complex. If there is a different reason, we can relay that on a case by case basis.